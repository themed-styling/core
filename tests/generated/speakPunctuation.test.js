import 'jest-styled-components'

import { speakPunctuation } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(speakPunctuation, 'speakPunctuation')
