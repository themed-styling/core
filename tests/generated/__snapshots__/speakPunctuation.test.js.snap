// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`speakPunctuation renders 0 1`] = `
Array [
  .c0 {
  speak-punctuation: 0;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: 0;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: 0;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders function 1`] = `
Array [
  .c0 {
  speak-punctuation: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: has;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: has;
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders invalid theme path 1`] = `
Array [
  .c0 {
  speak-punctuation: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders negative number 1`] = `
Array [
  .c0 {
  speak-punctuation: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  speak-punctuation: July;
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: July;
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    speak-punctuation: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    speak-punctuation: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    speak-punctuation: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    speak-punctuation: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  speak-punctuation: July;
}

@media May {
  .c0 {
    speak-punctuation: August;
  }
}

@media April {
  .c0 {
    speak-punctuation: September;
  }
}

@media January {
  .c0 {
    speak-punctuation: November;
  }
}

@media April {
  .c0 {
    speak-punctuation: August;
  }
}

@media June {
  .c0 {
    speak-punctuation: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: July;
}

@media May {
  .c0 {
    speak-punctuation: August;
  }
}

@media April {
  .c0 {
    speak-punctuation: September;
  }
}

@media January {
  .c0 {
    speak-punctuation: November;
  }
}

@media April {
  .c0 {
    speak-punctuation: August;
  }
}

@media June {
  .c0 {
    speak-punctuation: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    speak-punctuation: November;
  }
}

@media April {
  .c0 {
    speak-punctuation: January;
  }
}

@media January {
  .c0 {
    speak-punctuation: December;
  }
}

@media April {
  .c0 {
    speak-punctuation: April;
  }
}

@media June {
  .c0 {
    speak-punctuation: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    speak-punctuation: November;
  }
}

@media April {
  .c0 {
    speak-punctuation: January;
  }
}

@media January {
  .c0 {
    speak-punctuation: December;
  }
}

@media April {
  .c0 {
    speak-punctuation: April;
  }
}

@media June {
  .c0 {
    speak-punctuation: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    speak-punctuation: July;
  }
}

@media April {
  .c0 {
    speak-punctuation: August;
  }
}

@media January {
  .c0 {
    speak-punctuation: September;
  }
}

@media April {
  .c0 {
    speak-punctuation: November;
  }
}

@media June {
  .c0 {
    speak-punctuation: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    speak-punctuation: July;
  }
}

@media April {
  .c0 {
    speak-punctuation: August;
  }
}

@media January {
  .c0 {
    speak-punctuation: September;
  }
}

@media April {
  .c0 {
    speak-punctuation: November;
  }
}

@media June {
  .c0 {
    speak-punctuation: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    speak-punctuation: January;
  }
}

@media April {
  .c0 {
    speak-punctuation: December;
  }
}

@media January {
  .c0 {
    speak-punctuation: April;
  }
}

@media April {
  .c0 {
    speak-punctuation: May;
  }
}

@media June {
  .c0 {
    speak-punctuation: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    speak-punctuation: January;
  }
}

@media April {
  .c0 {
    speak-punctuation: December;
  }
}

@media January {
  .c0 {
    speak-punctuation: April;
  }
}

@media April {
  .c0 {
    speak-punctuation: May;
  }
}

@media June {
  .c0 {
    speak-punctuation: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders postitive number 1`] = `
Array [
  .c0 {
  speak-punctuation: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders string 1`] = `
Array [
  .c0 {
  speak-punctuation: July;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: July;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: June;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: June;
}

<div
    className="c0"
  />,
]
`;

exports[`speakPunctuation renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`speakPunctuation renders valid theme path 1`] = `
Array [
  .c0 {
  speak-punctuation: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  speak-punctuation: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
